OpenOversight/migrations/versions/2023-07-19-1638_1931b987ce0d_convert_timestamp_to_timestamptz.py

```diff
@@ -0,0 +1,129 @@
+"""convert timestamp to timestamptz
+
+Revision ID: 1931b987ce0d
+Revises: 07ace5f956ca
+Create Date: 2023-07-19 16:38:49.233825
+
+"""
+import os
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+from sqlalchemy.sql import func as sql_func
+
+
+# revision identifiers, used by Alembic.
+revision = "1931b987ce0d"
+down_revision = "07ace5f956ca"
+branch_labels = None
+depends_on = None
+
+TIMEZONE = os.getenv("TIMEZONE", "America/Chicago")
```

Comment:
I am assuming this `TIMEZONE` variable is used to correctly update the timestamps currently in the database. Is that correct?

Reply:
Exactly!